guides/moving-to-v2.md

```diff
@@ -57,19 +58,21 @@ coder workspaces list
 coder2 list
 ```
 
-
 ### Users
 
-If you 
+Like Coder v1, you can [enable SSO via OpenID Connect](https://coder.com/docs/coder-oss/latest/install/auth#step-2-configure-coder-with-the-openid-connect-credentials) so that any user in your federation can log in. Coder v2 optionally supports [GitHub (Enterprise)](https://coder.com/docs/coder-oss/latest/install/auth#step-1-configure-the-oauth-application-in-github) and [username/password](https://coder.com/docs/coder-oss/latest/users) authentication.
 
+> If you are interested in a bulk user and/or workspace migration utility, [we'd like to
+> hear from you](https://calendly.com/bpmct/30min).
+
+User-wide settings (e.g. avatar, shell, autostart times, dotfiles URL) are not currently supported in Coder v2 [(#3506)](https://github.com/coder/coder/issues/3506) but this behavior can often be replicated via workspace-level parameters. See the [feature comparion](#feature-comparison) below for more details.
 
 ### Workspaces
 
 To migrate Coder v1 workspaces, you'll need at least one [template](https://coder.com/docs/coder-oss/latest/templates)
 in your Coder v2 deployment, specifically with the image(s) you support in Coder v1.
 
-
-> If you are interested in a bulk workspace migration utility, [we'd like to
+> If you are interested in a bulk user and/or workspace migration utility, [we'd like to
 > hear from you](https://calendly.com/bpmct/30min).
```

Comment:
I do not like the 1:1, single-point-of-failure aspect of a calend.ly. Replace with support@coder.com so it lands in Slack and we have a record and can reply all to store the thread in Zendesk.